### PR TITLE
HackUTD IX Hackerpack

### PR DESCRIPTION
**Pull request by salmanjaher**

### CODE REVIEW

**Review by abdullah50053 — APPROVED** [review]

File: Hacker Pack · HackUTD.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+# Hacker Pack · HackUTD IX
```
Comment (adityarathod) [inline Hacker Pack · HackUTD.md:1]: why is this duplicated

**Review by adityarathod — CHANGES_REQUESTED** [review]
also delete .DS_Store

**Review by adityarathod — APPROVED** [review]
sexy

**Comment by abdullah50053** [comment]
Salman commit incoming 🚨  ⚠️

**Pull request merged**